Assets/MixedRealityToolkit/_Core/Definitions/Devices/TrackingState.cs

```diff
@@ -4,26 +4,33 @@
 namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices
 {
     /// <summary>
-    /// The Tracking State defines how a controller or headset is currently being tracked.
-    /// This enables developers to be able to handle non-tracked situations and react accordingly
+    /// The Tracking State defines how a device is currently being tracked.
+    /// This enables developers to be able to handle non-tracked situations and react accordingly.
     /// </summary>
+    /// <remarks>
+    /// Tracking is being defined as receiving sensor (positional and/or rotational) data from the device.
+    /// </remarks>
     public enum TrackingState
```

Comment:
I'd like to either use the new enum and get rid of this one, or not make a enum.

Reply:
Can you translate that please @StephenHodgson
I think for not, these changes should be fine and we can refine later once we have a working implementation and can evolve.

Reply:
I just think both of the Enums do the same job.  I find it a bit confusing about which to use.

Reply:
I can see removing TrackingState in favor of reporting PositionAccuracy and RotationAccuracy.

This would allow for Stephen's desire to have a hierarchical enum as well as keep positional and rotational tracking separate.

I can spin up a commit today.

Reply:
Thanks!

Reply:
Folding TrackingState and TrackingAccuracy is causing more churn than I had hoped... I'm working my way through it. It's not bad, just a little deeper than anticipated.

Reply:
Simplified this A LOT. By removing accuracy levels in favor of a bool (controller position only) for isApproximate.